src/main/java/com/teamabnormals/blueprint/core/endimator/interpolation/EndimationEasers.java

```diff
@@ -32,6 +46,136 @@ public final class EndimationEasers {
+	});
+	public static final Function<Float, Float> EASE_OUT_BACK = register("ease_out_back", progress -> {
+		float bounce = 1.70158F;
+		float base = bounce + 1.0F;
+		float base2 = progress - 1.0F;
+		return 1.0F + base * base2 * base2 * base2 + bounce * base2 * base2;
+	});
+	public static final Function<Float, Float> EASE_IN_OUT_BACK = register("ease_in_out_back", progress -> {
+		float bounce = 1.70158F;
+		float base = bounce * 1.525F;
+		if (progress < 0.5F) {
+			float base2 = 2 * progress;
+			return (base2 * base2 * ((base + 1) * 2 * progress - base)) * 0.5F;
+		} else {
+			float base2 = 2 * progress - 2;
+			return (base2 * base2 * ((base + 1) * (progress * 2 - 2) + base2) + 2) * 0.5F;
+		}
+	});
+	public static final Function<Float, Float> EASE_IN_ELASTIC = register("ease_in_elastic", progress -> {
+		float base = Mth.TWO_PI / 3.0F;
+		if (progress == 0.0F) {
+			return 0.0F;
+		} else if (progress == 1.0F) {
+			return 1.0F;
+		} else {
+			return (float) (-Math.pow(2, 10 * progress - 10) * Math.sin((progress * 10 - 10.75F) * base));
+		}
+	});
+	public static final Function<Float, Float> EASE_OUT_ELASTIC = register("ease_out_elastic", progress -> {
+		float base = Mth.TWO_PI / 3.0F;
+		if (progress == 0.0F) {
+			return 0.0F;
+		} else if (progress == 1.0F) {
+			return 1.0F;
+		} else {
+			return (float) (-Math.pow(2, -10 * progress) * Math.sin((progress * 10 - 0.75F) * base) + 1);
+		}
+	});
+	public static final Function<Float, Float> EASE_IN_OUT_ELASTIC = register("ease_in_out_elastic", progress -> {
+		float base = Mth.TWO_PI / 4.5F;
+		if (progress == 0.0F) {
+			return 0.0F;
+		} else if (progress == 1.0F) {
+			return 1.0F;
+		} else if (progress < 0.5F) {
+			return (float) (-(Math.pow(2, 20 * progress - 10) * Math.sin((20 * progress - 11.125F) * base)) * 0.5F);
+		} else  {
+			return (float) ((Math.pow(2, -20 * progress + 10) * Math.sin((20 * progress - 11.125F) * base)) * 0.5F + 1);
+		}
+	});
+	public static final Function<Float, Float> EASE_IN_BOUNCE = register("ease_in_bounce", progress -> 1.0F - bounce(1.0F - progress));
+	public static final Function<Float, Float> EASE_OUT_BOUNCE = register("ease_out_bounce", EndimationEasers::bounce);
+	public static final Function<Float, Float> EASE_IN_OUT_BOUNCE = register("ease_in_out_bounce", progress -> {
+		if (progress < 0.5F) {
+			return  (1.0F - bounce(1 - 2 * progress)) * 0.5F;
+		} else {
+			return (1.0F + bounce(2 * progress - 1)) * 0.5F;
+		}
+	});
+
+	private static float bounce(float progress) {
+		float base1 = 7.5625F;
+		float base2 = 2.75F;
+		if (progress < 1.0F / base2) {
+			return base1 * progress * progress;
+		} else if (progress < 2.0F / base2) {
+			return base1 * (progress -= 1.5F / base2) * progress + 0.75F;
+		} else if (progress < 2.5F / base2) {
+			return base1 * (progress -= 2.25F / base2) * progress + 0.9375F;
+		} else {
+			return base1 * (progress -= 2.625F / base2) * progress + 0.984375F;
+		}
+	}
```

Comment:
Move these lines to be below the constructor. Just a minor stylistic convention.